src/main/kotlin/pt/up/fe/ni/website/backend/model/dto/ProjectDto.kt

```diff
@@ -4,5 +4,6 @@ import pt.up.fe.ni.website.backend.model.Project
 
 class ProjectDto(
     val name: String,
-    val description: String
+    val description: String,
+    val isArchived: Boolean = false
```

Comment:
Keep in mind that this makes sense if it's ok to assume that if a project is being modified, then it's not supposed to be archived. Say, if a user updates the name of an archived project and does not pass `isArchived`, it will default to false and the project will be unarchived. If this is intended behavior, it's ok for me

Reply:
Good point. If, for some reason, it's not given in the edit endpoint, then I think the `isArchived` field should stay as it was. Should we add some logic to the edit endpoint or is there a more elegant way?

I'm happy to hear your thoughts on this @bdmendes @coutinho21